scanomatic/models/analysis_model.py

```diff
@@ -49,9 +48,33 @@ class VALUES(Enum):
     Cell_Estimates = 2
 
 
+class AnalysisModelFields(Enum):
```

Comment:
These explicit enums replaces the previous `FIELD_TYPE`, a bit more wordy but also helped me find some existing typos in validation functions.